launch/run.launch

```diff
@@ -2,13 +2,11 @@
   <arg name="device" default="/dev/ttyUSB0" />
   <arg name="rate" default="1000" />
   <arg name="sensor_frame_id" default="/sensor" />
-  <arg name="rvizconfig" default="$(find dynpick_driver)/launch/sample.rviz" />
```

Comment:
I can't find any web site that talks about this, but my own rule of thumb (and I believe this can be effective global ROS-wide) is to port as many variables as argument, so that once someone wants to `include` your launch file passing some args, s/he can immediately do so. 

So I suggest rolling back this change.